lib/main.dart

```diff
@@ -25,10 +21,16 @@ class MyApp extends StatelessWidget {
       ),
       initialRoute: '/',
       routes: {
-        // When navigating to the "/" route, build the FirstScreen widget.
+        // When navigating to the "/" route, build the LoginScreen widget.
         '/': (context) => LoginScreen(),
-        // When navigating to the "/second" route, build the SecondScreen widget.
-        '/second': (context) => HomeScreen(),
+        // When navigating to the "/home" route, build the HomeScreen widget.
+        '/home': (context) => HomeScreen(),
+        // When navigating to the "/login" route, build the LoginScreen widget.
+        '/login': (context) => LoginScreen(),
+        // When navigating to the "/add" route, build the Scan widget.
+        '/add': (context) => Scan(),
+        // When navigating to the "/search" route, build the SearchScreen widget.
+        //'/search' : (context) => SearchScreen()
```

Comment:
changed routes to match our program..I am guessing the new screen is going to be called search_screen